### ISSUE TITLE
Set searchField option

### ISSUE THREAD

**Issue by zooncc** [miguelcobain/ember-cli-selectize#64]
I want to search multiple field, so i add searchField property in controller:
```
export default Ember.Controller.extend({
    searchField: ["label", "value"]
});
```

and template:
```
{{ember-selectize
 selection=postId
 content=postOptions
 optionValuePath="content.id"
 optionLabelPath="content.value"
 multiple=false
 searchField=searchField
}}
```

but not working, it still only search `lable` field,  how to set searchField?
Thanks.

**Comment by miguelcobain** [comment]
Its not working because it isn't implemented.
We don't mention that in the README.

Suggestions are welcome.

**Comment by zooncc** [comment]
Thank you for your reply.
Can i request to add `searchField` option support?

**Comment by SirZach** [comment]
1 full year later and there's a PR!

**Comment by miguelcobain** [comment]
closed by #146
